src/JsonApi/Serializer/ErrorNormalizer.php

```diff
@@ -42,13 +43,17 @@ public function __construct(bool $debug = false, array $defaultContext = [])
         $this->defaultContext = array_merge($this->defaultContext, $defaultContext);
     }
 
-    public function normalize($object, $format = null, array $context = [])
+    public function normalize($object, string $format = null, array $context = [])
```

Comment:
Please do not change this.

Reply:
It's the cause of the job failure.

Reply:
ok I had symfony5 installed which has a changed the format.